apiserver/facades/client/sshclient/service.go

```diff
@@ -20,8 +20,11 @@ type ModelConfigService interface {
 type StubService interface {
 	// CloudSpec returns the cloud spec for the model.
 	CloudSpec(ctx context.Context) (cloudspec.CloudSpec, error)
+}
 
-	// GetExecSecretToken returns a token that can be used to run exec operations
+// ExecService will be replaced once the implementation is finished.
```

Comment:
Elaborate please.

Reply:
fark, this was a left over comment which i missed fixing